src/gwt/src/org/rstudio/core/client/layout/DualWindowLayoutPanel.java

```diff
@@ -510,10 +510,16 @@ private void unhookEvents()
       registrations_.removeHandler();
    }
 
+   public enum ReplaceMode
+   {
+      IF_DIFFERENT,
+      ALWAYS
+   }
    public void replaceWindows(LogicalWindow windowA,
-                              LogicalWindow windowB)
+                              LogicalWindow windowB,
+                              ReplaceMode replaceMode)
    {
-      if (windowA_ != windowA && windowB_ != windowB)
+      if (replaceMode == ReplaceMode.ALWAYS || (windowA_ != windowA && windowB_ != windowB))
```

Comment:
I know you didn't write the original code, but do you know why the comparisons between windows A & B are compared with `&&` as opposed to `||`? It seems like as long as one is different we want to do the replace.

Reply:
You added that comparison in this commit:

11367597c893c239b6c0acde6a4e9d56e49e9905

Reply:
Wow ok, I think this is a bug and should be changed.

Reply:
Per discussion I have removed that comparison completely. Thanks!